docs/ref/templates/builtins.txt

```diff
@@ -957,6 +957,83 @@ output (as a string) inside a variable. This is useful if you want to use
     {% now "Y" as current_year %}
     {% blocktranslate %}Copyright {{ current_year }}{% endblocktranslate %}
 
+.. templatetag:: partial
+
+``partial``
+-----------
+
+.. versionadded:: 6.0
+
+Renders a partial that was defined with :ttag:`partialdef`.
+
+Usage:
+
+.. code-block:: html+django
+
+    {% partial partial_name %}
+
+The ``partial_name`` argument is the string identifier of the partial to render.
+
+Example:
+
+.. code-block:: html+django
+
+    {% partialdef button %}
+        <button class="btn">{{ button_text }}</button>
+    {% endpartialdef %}
+
+    {% partial button %}
+    {% partial button %}
+
+
+.. templatetag:: partialdef
+
+``partialdef``
+--------------
+
+.. versionadded:: 6.0
+
+Defines a reusable template partial that can be rendered multiple times within
```

Comment:
The word "multiple" here may imply that a user **** to use the partial multiple times. Could we soften this up to be more inviting and less strict?
```suggestion
Defines a reusable template fragment that can be rendered multiple times within
```